packages/@okta/vuepress-site/docs/guides/protect-your-api/main/index.md

```diff
@@ -4,186 +4,237 @@ excerpt: Configure your Okta org and your server-side application to secure your
+Earlier you [noted your authorization server name and audience](#note-your-authorization-server-name-and-audience). Add these and your Okta domain to your API's configuration.
 
-<StackSnippet snippet="configmid" />
+<StackSnippet snippet="configureapi" />
 
-## Create your REST endpoints
+## Create two endpoints to secure
 
-Create new REST endpoints in your application that cover two different use cases:
+Create two endpoints in your project that cover two different use cases:
 
 * `api/whoami`&mdash;a protected endpoint (access-restricted API)
 * `api/hello`&mdash;an endpoint that anonymous users can access (unsecured API)
 
-<StackSnippet snippet="createroute" />
+<StackSnippet snippet="createendpoints" />
 
-## Configure required authentication
+## Configure different levels of access for different endpoints
 
-In many APIs, all the endpoints require authorization. In others, there may be a mix of protected and unprotected (anonymous) endpoints. These examples show you how to do both.
+In many APIs, all endpoints require authorization. There may be a mix of protected and unprotected endpoints in others. These examples show you how to assign protected and unprotected access to an endpoint.
 
-### Require authorization for everything
+### Require authorization for all endpoints
 
 <StackSnippet snippet="reqautheverything" />
 
 ### Allow anonymous access for specific routes
 
-To allow a mix of protected and unprotected (anonymous) endpoints, configure access on a per route basis.
+Configure access on a per-route basis to allow a mix of protected and anonymous endpoints.
 
 <StackSnippet snippet="reqauthspecific" />
 
-### Configure CORS if necessary
+### Enable CORS for your API
 
-Configuring Cross-Origin Resource Sharing (CORS) is only required if the API is being called from a browser app hosted on a different domain. For example, if you're hosting a single-page JavaScript app at `example.com`, which consumes an API endpoint hosted on `api.example.com`, you need to enable CORS.
+Enable [Cross-Origin Resource Sharing  (CORS)](https://fetch.spec.whatwg.org/#http-cors-protocol) only if the API is being called from an application or API hosted on a different domain. For example, if your API is hosted on `api.example.com` while your application is accessing it from `example.com`, you must enable CORS.
 
 <StackSnippet snippet="configcors" />
 
-## Consume your secure API endpoints
+## Test your API is secure
 
-Test your APIs with an access token that is sent to your endpoint for validation.
+You can now test if your endpoint security works as intended. To do this, you will:
 
-### Use an access token with your API endpoint
+1. [Create an API services integration to represent another machine or service attempting to make requests to the API](#create-an-api-services-integration)
```

Comment:
There's no intro statement here so the lack of periods isn't as problematic; the list just reads like a sequence of links (which is fine!)

Reply:
@grahamsmith-okta Not sure why you didn't see it, but there is an intro sentence here that now reads as follows

You can now test if your endpoint security works as intended. To do this, complete the following steps: